packages/jest-runtime/src/helpers.ts

```diff
@@ -53,3 +54,12 @@ export const findSiblingsWithFileExtension = (
 
   return '';
 };
+
+export const printRequireStack = (
+  requireStack: Array<string>,
+  rootDir: Path,
+): string => `
+
+Require stack:
+  ${requireStack.map(p => p.replace(`${rootDir}${path.sep}`, '')).join('\n  ')}
```

Comment:
we have a helper called `replaceRootDirInPath` in `jest-config`, please use it

Reply:
also, please pass the path through `slash` to normalize path separators to unix style (we do this for most paths shown to the user already)